articles/multifactor-authentication/developer/libraries/android/index.md

```diff
@@ -0,0 +1,148 @@
+---
+toc_title: Getting Started with Guardian for Android
+url: /multifactor-authentication/developer/libraries/android
+description: Installation, usage, and configuration options guide for Guardian for Android
+---
+
+# Guardian for Android
+The Guardian for Android Software Development Kit allows developers to integrate their native Android apps with Guardian functionality, providing easy and secure access to multifactor authentication with push notifications.
```

Comment:
I don't know if this is clear enough. It allows to "make their own guardian", transforming their apps on the second factor itself.
Integration with guardian sounds like you can add support to use guardian on lock for example